bin/i18n/sync-codeorg-all.rb

```diff
@@ -183,6 +187,7 @@ def main
     sync_out if should_i "sync out"
     create_down_out_pr if options[:with_pull_request]
     upload_i18n_stats if should_i "upload translation stats"
+    checkout_staging if should_i "return to the staging branch"
```

Comment:
Should we do this before the `sync in` as well to prevent the "you must be on the staging branch" message?

Reply:
sure!